runtime/rasdump/javadump.cpp

```diff
@@ -3378,6 +3381,32 @@ JavaCoreDumpWriter::writeTrailer(void)
 {
 	_OutputStream.writeCharacters(
 		"0SECTION       Javadump End section\n"
+		"NULL           =================================\n"
+	);
+
+	PORT_ACCESS_FROM_JAVAVM(_VirtualMachine);
+	OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
+
+	/* Write the date and time that the dump completed generation. */
+	char timeStamp[_MaximumTimeStampLength + 1];
+	int64_t now = j9time_current_time_millis();
+
+	omrstr_ftime_ex(timeStamp, _MaximumTimeStampLength, TIMESTAMP_FORMAT, now, OMRSTR_FTIME_FLAG_LOCAL);
+	timeStamp[_MaximumTimeStampLength] = '\0';
+
+	_OutputStream.writeCharacters("1TIDMPCOMPLETE Dump Completed Date: ");
+	_OutputStream.writeCharacters(timeStamp);
+	_OutputStream.writeInteger(now % 1000, ":%03d"); /* Add the milliseconds. */
+	_OutputStream.writeCharacters("\n");
+
+	/* Calculate the approximate time it took to produce the dump. */
+	int64_t duration = now - _DumpStart;
+
+	_OutputStream.writeCharacters("1TIDMPDURATION Approximate time to produce the dump (ms): ");
```

Comment:
I don't think we need to capitalize each word in the `1TIDMPCOMPLETE` line, just "Dump", for consistency with `1TIDMPDURATION`. Also, I think the `(ms)` before the time should be removed, and replaced by `ms` after it. Here's a sample of what I think the output should be:
```
1TIDMPCOMPLETE Dump completed: 2022/09/26 at 14:40:47:718
1TIDMPDURATION Approximate time to produce this dump: 4 ms
```